samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -122,22 +123,44 @@
   },
 
   //
-  // CORS configuration
+  // Planner can determine which skill functions, if any, need to be used to fullfill a user's request.
+  // https://learn.microsoft.com/en-us/semantic-kernel/concepts-sk/planner
+  //
+  "Planner": {
+    "MaxTokens": 1024,
+    "RelevancyThreshold": 0.78
+  },
+
+  //
+  // CORS
   //
   "AllowedHosts": "*",
   "AllowedOrigins": [
-    "http://localhost:3000" // The frontend application
+    "http://localhost:3000"
   ],
 
+  //
+  // Server endpoints
+  //
+  "Kestrel": {
+    "Endpoints": {
+      "Https": {
+        "Url": "https://localhost:40443"
+      }
+    }
+  },
+
   //
   // Logging configuration
   //
   "Logging": {
     "LogLevel": {
       "Default": "Warning",
+      "SemanticKernel.Service": "Information",
+      "Microsoft.SemanticKernel": "Information",
       "Microsoft.AspNetCore.Hosting": "Information",
       "Microsoft.Hosting.Lifetime": "Information",
-      "Microsoft.SemanticKernel": "Information"
+      "Microsoft.AspNetCore.Server": "Error"
```

Comment:
That's pretty high.  We don't want warnings?

Reply:
Visual Studio's default launch settings set Urls and was causing a warning to be thrown. I found another way around it, so I'll remove this line.